ios/brave-ios/Tests/ClientTests/URLFormatTests.swift

```diff
@@ -89,12 +100,14 @@ import XCTest
     checkInvalidURL("创业咖啡")
     checkInvalidURL("创业咖啡 中国")
     checkInvalidURL("创业咖啡. 中国")
-    checkInvalidURL("data:text/html;base64,SGVsbG8gV29ybGQhCg==")
```

Comment:
No invalid anymore ?

Reply:
This gets submitted to the search engine automatically.